src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/response/model/Prisoners.kt

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.response.model
+
+import java.time.LocalDate
+
+
+data class Prisoner(
```

Comment:
Is it worth putting this in a directory called `PrisonerSearch` like they do on the Approved Premises project [here](https://github.com/ministryofjustice/hmpps-approved-premises-api/tree/main/src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/model/prisonsapi). So we'd have `/hmppsaccreditedprogrammesapi/model/prisonerSearch/Prisoner.kt` I believe it should be singular too?

This would be a useful pattern for separating models in our own service from those from other APIs and grouping response types clearly.